diff --git a/base/BUILD.gn b/base/BUILD.gn
index e8dfcec63428fbf61f1ec3ab71fce7af0a8d52f4..7243c2e0eee933fefd107cace3fbeb13a788344c 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4743,6 +4743,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/UserDataHost.java",
       "android/java/src/org/chromium/base/ValueChangedCallback.java",
     ]
+    sources += brave_java_base_sources
 
     resources_package = "org.chromium.base"
 
@@ -5381,6 +5382,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/shared_preferences/SharedPreferencesManager.java",
       "android/java/src/org/chromium/base/shared_preferences/StrictPreferenceKeyChecker.java",
     ]
+    sources += brave_base_shared_preferences_java_sources
   }
 
   android_library("flagged_apis_java") {
