diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index e0e28ecc47c802e490ff0a66efe67b4ac4d52f12..721f726fe1ee31e8a8932b9557923fdc8a49038e 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -5200,6 +5200,7 @@ const FeatureEntry::FeatureVariation kContextualTaskEntryPointVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
