diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index c8b07525003cca78899322cd2c87b57b6ed2130c..bc11265669da225af2b85b4521fbb170fce6b446 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3700,6 +3700,7 @@ source_set("browser") {
   if (is_chromeos) {
     deps += [ "//ash/constants" ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
